cmd/bff/main.go

```
@@ -171,18 +174,38 @@ func checkDownstreamHealth(cfg *config.Config, logger *log.Logger) bool {
 	}
 
-	// Try to connect to the service (TCP check)
-	resp, err := client.Get(serviceURL + "/health")
+	// Build health check URL by replacing the port in the base URL
+	healthURL := replacePort(baseURL, healthPort) + "/health"
+
+	resp, err := client.Get(healthURL)
 	if err != nil {
 		return false
 	}
 	defer resp.Body.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)